torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -10,18 +10,63 @@
 from .bert_pytorch.model import BERT
 from torch.utils.data import DataLoader
 
-torch.manual_seed(1337)
-random.seed(1337)
-np.random.seed(1337)
-torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.deterministic = False
```

Comment:
I guess setting `cudnn.deterministic` to `False` helps improve GPU utilization numbers? How much does it help?

Reply:
I thought we agreed that deterministic was only to be used when there was a reason. We want the cudnn to be able to run in its best mode which is unconstrained.

Reply:
Gotcha!